src/main/java/edu/hawaii/its/api/service/OotbGroupingPropertiesService.java

```diff
@@ -130,56 +141,159 @@ public GetGroupsResults getGroupsResults() {
         return getGroupsResults;
     }
 
-    /* Updating Member(s) */
+    public Boolean updateActiveUserProfile(List<String> authorities, String uid, String uhUuid, String name, String givenName) {
+        updateActiveUserProfileInGroupingsOotbs(authorities, uid, uhUuid, name, givenName);
+        updateActiveUserProfileInGetGroupResultsBean(authorities, uid, uhUuid, name, givenName);
+        updateActiveUserProfileInHasMembersResultsBean(authorities, uid, uhUuid, name, givenName);
+        addActiveUserProfileToSubjectListWithoutDuplicate(authorities, uid, uhUuid, name, givenName);
+        addActiveUserProfileToMembersListWithoutDuplicate(authorities, uid, uhUuid, name, givenName);
+        return true;
+    }
 
-    public void ootbRemoveMember(String currentUser, String groupPath, String uhIdentifier) {
+    public void addActiveUserProfileToSubjectListWithoutDuplicate(List<String> authorities, String uid, String uhUuid, String name, String givenName) {
+        WsGetSubjectsResults wsGetSubjectsResults = getSubjectsResults().getWsGetSubjectsResults();
 
-        WsGetMembersResult[] updatedWsGetMembersResults = Arrays.stream(wsGetMembersResultsList())
+        WsSubject defaultUserSubject = new WsSubject();
+        defaultUserSubject.setName(givenName);
+        defaultUserSubject.setIdentifierLookup(uid);
+        defaultUserSubject.setId(uhUuid);
+        defaultUserSubject.setResultCode("SUCCESS");
+        defaultUserSubject.setAttributeValues(
+                new String[] { uid, "", "LASTNAME", "FIRSTNAME" });
```

Comment:
I'm not sure how you define all the users, but could you hardcode the default MEMBER, OWNER, and ADMIN users into the data harness? Then we could reduce a lot of this `new WsSubject()` code.